core/json/json_parser.h

```diff
@@ -28,8 +28,11 @@ class JsonParser {
  public:
   static std::optional<JsonParser> Parser(const std::string& path) noexcept;
   explicit JsonParser(std::ifstream& f);
+  explicit JsonParser(nlohmann::json json);
+  static std::optional<JsonParser> Parser(nlohmann::json obj) noexcept;
```

Comment:
Po co ??

Reply:
![image](https://github.com/Simba-Avionic/srp/assets/59980471/849052d7-868b-418a-973d-ea65912af637)
lepiej mi to tutaj pasowało